### ISSUE TITLE
stared/livelossplot (124682643)

### ISSUE THREAD

**Issue by gfleetwood** [gfleetwood/asteres#1429]
https://github.com/stared/livelossplot

Live training loss plot in Jupyter Notebook for Keras, PyTorch and others
